addons/docs/src/blocks/enhanceSource.ts

```diff
@@ -11,14 +11,20 @@ interface StorySource {
   locationsMap: { [id: string]: { startBody: Location; endBody: Location } };
 }
 
+/**
+ * Replaces full story id name like: story-kind--story-name -> story-name
+ * @param id
+ */
+const storyIdToSanitizedStoryName = (id: string) => id.replace(/^.*?--/, '');
```

Comment:
I don't like this function but we can't use context.name as it can be overriden by CSF. The id is the only place that contains original sanitized name but it also contains story title (before -- double dash). 
So I simply remove sanitized story title from id and get sanitized name